tox.ini

```diff
@@ -31,7 +31,7 @@ deps =
 
 [testenv:docs]
 basepython =
-    python2.7
+    python3.6
```

Comment:
I'll second a comment @mgedmin made about using 3.7 specifically in one environment and 3.6 in another. The `coverage` environment is running Sphinx, so if we're concerned about Sphinx compatibility with 3.7 we probably shouldn't move the coverage environment that far. OTOH, it seems to work fine, so for simplicity maybe they should just match.

Reply:
After thinking about it for two days, I also think the _virtual_ problems of choosing 3.7 don't overweight this odd look.